src/cpp/session/SessionModuleContext.cpp

```diff
@@ -3005,6 +2996,23 @@ Error adaptToLanguage(const std::string& language)
    return Success();
 }
 
+std::string rstudioVersion(bool normalizeSuffix)
```

Comment:
This was originally in the RMarkdown session file, but since it wasn't really an RMarkdown-specific function I pulled that out here for re-use.